src/main/java/org/sonarlint/intellij/config/global/rules/RulesFilterModel.java

```diff
@@ -101,10 +101,10 @@ public void reset(boolean triggerListener) {
   }
 
   public boolean filter(RulesTreeNode.Rule rule) {
-    if (showOnlyEnabled && !rule.isActivated()) {
+    if (showOnlyEnabled && Boolean.FALSE.equals(rule.isActivated())) {
```

Comment:
I find it strange that `isActivated` returns the wrapper type, but it was already like this. Do you know if it can actually happen that it is null?

Reply:
For a `RulesTreeNode.Rule` it doesn't happen in the code, but for `RulesTreeNode.Language` is can

So in this case it's 'safe' because it's only Rule